lib/index_shotgun/cli.rb

```diff
@@ -0,0 +1,39 @@
+require "thor"
+require "index_shotgun"
+require "active_support/core_ext/hash"
+
+module IndexShotgun
+  class CLI < Thor
+    desc "mysql", "Search duplicate indexes on mysql"
+    option :database, required: true
+    option :encoding, default: "utf8"
+    option :pool, default: 5, type: :numeric
+    option :host, default: "localhost"
+    option :post, default: 3306, type: :numeric
+    option :username
+    option :password
+    def mysql
+      analyze("mysql2")
+    end
+
+    desc "version", "Show index_shotgun version"
+    def version
+      puts IndexShotgun::VERSION
+    end
+
+    private
+
+      def analyze(adapter)
+        begin
+          require adapter
+        rescue LoadError
+          puts "[ERROR] #{adapter} is not installed. Please run `gem install #{adapter}` and install gem"
```

Comment:
Line is too long. [105/80]<br>Do not write to stdout. Use Rails' logger if you want to log.